frontend_tests/casper_tests/06-settings.js

```diff
@@ -99,6 +90,15 @@ casper.then(function () {
     });
 });
 
+casper.then(function () {
+    casper.waitUntilVisible('#account-settings-status', function () {
+        casper.test.assertSelectorHasText('#account-settings-status', 'Updated settings!');
+
+        casper.click('[data-section="your-bots"]');
```

Comment:
I'm just surprised this passes with the string `your-bots` instead of `your-account`.  Do you know why this is happening?  Is `your-bots` the correct string here?